templates/repo/issue/milestone_issues.tmpl

```diff
@@ -215,7 +215,8 @@
 						{{end}}
 					{{end}}
 
-					<a class="title" href="{{$.RepoLink}}/issues/{{.Index}}">{{.Title | RenderEmoji}}</a>
+					{{ $issueLink:= printf "%s/issues/%d" $.RepoLink .Index }}
+					<span class="title">{{RenderCommitMessageLink .Title $.RepoLink $issueLink $.Repository.ComposeMetas}}</span>
```

Comment:
Very minor but I think a sub-template (or however those are called) could achieve the same without the extra variable):

````handlebars
{{RenderCommitMessageLink .Title $.RepoLink (printf "%s/issues/%d" $.RepoLink .Index) $.Repository.ComposeMetas}}